src/toil/jobGraph.py

```diff
@@ -133,7 +143,7 @@ def restartCheckpoint(self, jobStore):
             jobStore.update(self) # Update immediately to ensure that checkpoint
             # is made before deleting any remaining successors
 
-            if len(self.stack) > 0 or len(self.services) > 0:
+            if self.stack or self.services:
```

Comment:
Why change this here and not up at 136?

Reply:
I just missed it.  Thanks for the catch!

Reply:
Changed 136.